### PR TITLE
Swift 4

### PR DESCRIPTION
**Pull request by stephencelis**
Chatted with the fine folks on the SPM Slack and looks like this is the way to go about supporting Swift 4 and 5 at the same time.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:4.2
```
Comment (stephencelis) [inline Package.swift:1]: The tools version should (almost) always be the minimum supported version. We could alternately maintain a separate manifest as described here: https://github.com/apple/swift-package-manager/blob/master/Documentation/Usage.md#version-specific-manifest-selection

But I was discouraged from doing so.

**Comment by stephencelis** [comment]
Once merged this should fix #53.

**Comment by stephencelis** [comment]
(Build's broken because SnapshotTesting dropped Swift 4 support. I'll add it back.)

**Review by mbrandonw — APPROVED** [review]
Nice!

**Pull request merged**